nsv13/code/modules/overmap/weapons/damage.dm

```diff
@@ -43,6 +43,11 @@ Bullet reactions
 				return BULLET_ACT_FORCE_PIERCE // :)
 			else
 				return FALSE //Shields absorbed the hit, so don't relay the projectile
+		if(shield_result == SHIELD_NOEFFECT && !failure_sound_cooldown)
```

Comment:
Okay, _now_ I have a comment on the code. If I'm reading this right, it looks like it will play the shields down effect at the same time it would play the projectile hitting sound with a cooldown of 5 seconds. 

Now, I'm no audio engineer, but that sounds like adding another sound that doesn't tell you something new (the sound of the projectile striking the hull is very obvious, and in a lot of cases, excessively loud), is just a bad idea.

Reply:
I personally think it's nice to have the little extra warning- it can be hard to tell sometimes, but I could look into moving the trigger into when the shield's charge is dropped to zero- but that'll play even when you aren't actually taking damage

Reply:
I have nearly 300hrs in engineering and probably a solid 20hrs of that standing at that shield generator. It is _never_ at zero for more than a moment unless the power requested is higher than the power provided. 

_It is also extremely obvious to everyone when the shield are down because the shield deflect sounds stop playing and the hull hit effects start._

If you want to add in a warning effect for when the shields are at 33% of effective strength or less with an appropriate cooldown, that would be useful- and would not be attached to actual projectile hits. _**It would belong in the shield generator dm file and not in damage.dm**_

Reply:
Even more importantly, this sound effect will also trigger with the PDSR, as this is damage.dm and the PDSR uses this sound for another reason. So you're going to have that PDSR sound playing for 2 separate reasons on the Galactica.

Reply:
fine whatever. I'll just close this then